src/pages/my-offers/MyOffers.tsx

```diff
@@ -0,0 +1,141 @@
+import { useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
```

Comment:
remove this space and add it before hooks

Reply:
removed